docs/help/_posts/2013-04-15-compat-args.markdown

```diff
@@ -3,22 +3,13 @@ title: Compatibility argument matchers
 layout: post
 ---
 
-NSubstitute [argument matchers](/help/argument-matchers) depend on having C# 7.0 or later (as of NSubstitute 4.0). If you are stuck on an earlier version of C# you may get an error like the following when trying to use a matcher like `Arg.Is(123)`:
+NSubstitute [argument matchers](/help/argument-matchers) depend on having C# 7.0 or later (as of NSubstitute 4.0). This lets them be used with `out` and `ref` parameters, but it also means that if you are stuck on an earlier version of C# you may get an error like the following when trying to use a matcher like `Arg.Is(123)`:
 
 > `CS7085: By-reference return type 'ref T' is not supported.`
 
 If you have C# 7.0-compatible tooling installed you can [set `<LangVersion />` in your test csproj file](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version) to `7` or higher, or to `latest` or `default`.
 
-Stuck with pre-7.0 tooling? Then use `CompatArg` from the `NSubstitute.Compatibility` namespace instead of `Arg`. `CompatArg` will work everywhere `Arg` does, with the exception of matching `out` and `ref` args (which require C# 7.0 features). For example, if the documentation mentions `Arg.Is(42)`, you can instead use `CompatArg.Is(42)`. 
-
-## Using `CompatArg`
-
-First, add `NSubstitute.Compatibility` to the list of imports:
-
-    using NSubstitute;
-    using NSubstitute.Compatibility;
-
-Then use `CompatArg` in place of `Arg`:
+Stuck with pre-7.0 tooling? Then use `Arg.Compat` instead of `Arg`, or use `CompatArg` in the `NSubstitute.Compatibility` namespace. `Arg.Compat` will work everywhere `Arg` does, with the exception of matching `out` and `ref` args. For example, if the documentation mentions `Arg.Is(42)`, you can instead use `Arg.Compat.Is(42)`. `CompatArg` is a bit trickier to setup, but will make migrating between `Arg` and `Arg.Compat` easier. Both options are described below.
```

Comment:
I'd rephrase to `... is a bit trickier to setup, but sometimes might make a migration easier.` (or similar) to not sell it as a silver bullet  :) It might help, but if you don't have base class for all your tests (for instance, we never have those one in our tests), then it will be much easier to replace `Arg.` to `Arg.Compat.`.